### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by l5h5t7**

### CODE REVIEW

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -12,23 +12,23 @@
         "description": ""
     },
     "options_remove_origin_confirm": {
-        "message": "Are you sure you want to remove this origin from Privacy Badger?",
+        "message": "你确定要从隐私獾中移除这个源吗？",
         "description": ""
     },
     "badger_status_noaction": {
-        "message": "No tracking for ",
+        "message": "没有追踪",
```
Comment (ghostwords) [inline src/_locales/zh/messages.json]: Should have a trailing space here.

File: src/_locales/zh/messages.json [hunk 2]
```diff
@@ -172,7 +172,7 @@
         "description": "Beginning of message on the Tracking Domains options page tab that shows the total number of trackers detected so far."
     },
     "social_tooltip_button": {
-        "message": " button.",
+        "message": "按钮。",
```
Comment (ghostwords) [inline src/_locales/zh/messages.json]: Should have a leading space here.

**Comment by l5h5t7** [comment]
I am sorry for my carelessness and I committed a change to fix the issues you mentioned.

File: src/js/multiDomainFirstParties.js [hunk 3]
```diff
@@ -306,17 +306,6 @@ var multiDomainFirstPartiesArray = [
   ],
   ["gotomeeting.com", "citrixonline.com"],
   ["guardian.co.uk", "guim.co.uk", "guardianapps.co.uk", "theguardian.com", "gu-web.net"],
-  [
```
Comment (ghostwords) [inline src/js/multiDomainFirstParties.js]: Looks like this file got updated by accident.

**Comment by ghostwords** [comment]
No problem, thank you for your contributions!

File: src/js/multiDomainFirstParties.js (hunk 3 above)
Comment (l5h5t7) [reply src/js/multiDomainFirstParties.js]: Fixed.

**Comment by ghostwords** [comment]
Thank you!

**Pull request merged**